Library/Formula/mydumper.rb

```diff
@@ -0,0 +1,24 @@
+class Mydumper < Formula
+  desc "How MySQL DBA & support engineer would imagine 'mysqldump' ;-)"
+  homepage "https://launchpad.net/mydumper"
+  url "https://launchpad.net/mydumper/0.6/0.6.2/+download/mydumper-0.6.2.tar.gz"
+  version "0.6.2"
+  sha256 "fa28563e8967752828954c5d81e26ef50aad9083d50a977bf5733833b23e3330"
+
+  depends_on "cmake" => :build
+  depends_on "sphinx" => :python
```

Comment:
Don't do this. You can use resources, which is a much nicer experience for the user.

Barring the unrelated comment on circular deps in [this block](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/cmake.rb#L19-L87), you can pretty much copy that directly, including the `option "without-docs", "Don't build man pages"` line above.